### PR TITLE
mg-ASMFT-288

### PR DESCRIPTION
**Pull request by Gregorma**
Adding Combo LS & IDS to cart with BD info

### CODE REVIEW

File: demo/checkout_mods.js [hunk 1]
```diff
@@ -118,6 +129,20 @@ var corpIdsSelectRegionPlanBD = function () {
   click('button.ge-btn--action:nth-child(3)')
 }
 
+var caCorpLsSelectingRegionPlanBD = function () {
+  click('#select_4')
+  scroll('md-option[value="AB"')
+  click('button.ge-btn.ge-btn--action.btn__addPlan')
+	click('.identity__header > h3:nth-child(1)')
```
Comment (TrentonWalkup) [inline demo/checkout_mods.js]: indention

File: demo/checkout_mods.js [hunk 2]
```diff
@@ -377,6 +419,20 @@ var bdPaymentInfo = function () {
...
+var caBdPaymentInfo = function () {
+  browser.pause(500)
+  click('li.uib-tab:nth-child(2) > a:nth-child(1) > uib-tab-heading:nth-child(1)')
+  browser.pause(500)
+  click('#radio_1 > div.md-container.md-ink-ripple > div.md-off')
+  enterText('//input[@name="payor_name"]', 'Automated Test')
+  enterText('//input[@name="bank_transit_number"]', '06069')
+  enterText('//input[@name="bank_institution_number"]' , '010')
+  enterText('//input[@name="bank_account_number"]', '9113711')
+  browser.pause(5000)
```
Comment (TrentonWalkup) [inline demo/checkout_mods.js]: can change this back to 500ms

File: demo/checkout_mods.js [hunk 3]
```diff
@@ -354,6 +395,7 @@ var askOptions = function () {
 
 //5.) Payments
 var bankName = 'VISION BANK NA'
+var caBankName = 'Canadian Imperial Bank Of Commerce'
```
Comment (TrentonWalkup) [inline demo/checkout_mods.js]: don't need

Comment (roppel5ls) [reply demo/checkout_mods.js]: If you can input a bank name, I would leave this and use it to input the bank name field

**Pull request closed without merging**